### PR TITLE
[FrontendBundle] allow usage of auto-wired Frontend Controllers

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  |no
| BC breaks?    | no
| Deprecations? |no
| Fixed tickets | #1139

### CODE REVIEW

File: src/CoreShop/Bundle/FrontendBundle/DependencyInjection/CompilerPass/RegisterFrontendControllerPass.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                    $controllerDefinition->setArguments([
+                        new Reference('security.authentication_utils'),
+                        new Reference('form.factory'),
+                        new Reference('coreshop.context.shopper'),
+                    ]);
+                break;
+
+                case 'payment':
+                    $controllerDefinition->setMethodCalls([
+                        ['setContainer', [new Reference('service_container')]]
```
Comment (solverat) [inline src/CoreShop/Bundle/FrontendBundle/DependencyInjection/CompilerPass/RegisterFrontendControllerPass.php]: already defined at line 33?

Comment (solverat) [inline src/CoreShop/Bundle/FrontendBundle/DependencyInjection/CompilerPass/RegisterFrontendControllerPass.php]: already defined at line 42?

File: src/CoreShop/Bundle/FrontendBundle/Resources/config/services.yml [hunk 2]
```diff
@@ -8,94 +8,94 @@ services:
...
+#        class: '%coreshop.frontend.controller.payment%'
+#        public: true
+#        arguments:
+#            - '@coreshop.order.payment_provider'
+#            - '@coreshop.repository.order'
+#            - '@coreshop.factory.payum_get_status'
+#            - '@coreshop.factory.payum_resolve_next_route'
+#            - '@coreshop.factory.payum_confirm_order'
+#        calls:
+#            - [setContainer, ['@service_container']]
```
Comment (solverat) [inline src/CoreShop/Bundle/FrontendBundle/Resources/config/services.yml]: why not removing it completely?

File: src/CoreShop/Bundle/FrontendBundle/DependencyInjection/CompilerPass/RegisterFrontendControllerPass.php (hunk 1 above)
Comment (dpfaffenbauer) [reply src/CoreShop/Bundle/FrontendBundle/DependencyInjection/CompilerPass/RegisterFrontendControllerPass.php]: line 42 also sets the TemplateConfiguration, but the PaymentController is not a FrontendController

File: src/CoreShop/Bundle/FrontendBundle/Resources/config/services.yml (hunk 2 above)
Comment (dpfaffenbauer) [reply src/CoreShop/Bundle/FrontendBundle/Resources/config/services.yml]: done

**Pull request merged**